src/cheri_sys_regs.sail

```diff
@@ -3,22 +3,33 @@
 /* Capability control csr */
 
 bitfield ccsr : xlenbits = {
+  ugclg   :  3,      /* Global Cap Load Gen bit for User pages */
+  sgclg   :  2,      /* Global Cap Load Gen bit for System (not User) pages */
   d       :  1,      /* dirty  */
   e       :  0       /* enable */
 }
 
-register mccsr : ccsr
-register sccsr : ccsr
+register sccsr : ccsr /* mccsr aliases sccsr */
 register uccsr : ccsr
 
 /* access to CCSRs */
 
-// for now, use a single privilege-independent legalizer
-function legalize_ccsr(c : ccsr, v : xlenbits) -> ccsr = {
+function legalize_ccsr(p : Privilege, c : ccsr, v : xlenbits) -> ccsr = {
```

Comment:
Maybe `cp`? Otherwise it could be confused for what privilege the hart is currently running with.

Reply:
I went with the slightly longer `csrp` since `cp` might still be read as `current privilege` at a glance.